dlrover/examples/deepctr_job_auto_scale.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: elastic.iml.github.io/v1alpha1
+kind: ElasticJob
+metadata:
+  name: deepctr-sample-auto-scaling-114
```

Comment:
Rename `deepctr-sample-auto-scaling-114` to `deepctr-auto-scaling-job`.

The file name `dlrover/examples/deepctr_job_auto_scale.yaml`  -> `dlrover/examples/deepctr_auto_scale_job.yaml`